README.md

```diff
@@ -172,6 +172,8 @@ For documentation on the latest development code see the [documentation index][d
 * [haproxy](./plugins/inputs/haproxy)
 * [hddtemp](./plugins/inputs/hddtemp)
 * [httpjson](./plugins/inputs/httpjson) (generic JSON-emitting http service plugin)
+* [icecast](./plugins/inputs/icecast)
+* [internal](./plugins/inputs/internal)
```

Comment:
I think this should not belong to this PR. Maybe submit a separate one in case you detect missing plugins in that list.